go/signedexchange/README.md

```diff
@@ -69,26 +69,19 @@ Here, we assume that you have an access to an HTTPS server capable of serving st
     - Note: If you are using [Firebase Hosting](https://firebase.google.com/docs/hosting/) as your HTTPS server, see an example config [here](https://github.com/WICG/webpackage/blob/master/examples/firebase.json).
 
 1. Navigate to the signed exchange URL using a web browser supporting signed exchanges.
-    - As of July 2018, you can use Chrome M69 [Dev](https://www.google.com/chrome/?extra=devchannel)/[Canary](https://www.google.com/chrome/browser/canary.html) versions with a command-line flag to enable signed exchange support.
-      ```
-      # Launch chrome dev set to ignore certificate errors of the self-signed certificate,
-      # with an experimental feature of signed exchange support enabled.
-      google-chrome-unstable \
-        --user-data-dir=/tmp/udd \
-        --ignore-certificate-errors-spki-list=`openssl x509 -noout -pubkey -in cert.pem | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | base64` \
-        --enable-features=SignedHTTPExchange \
-        https://yourcdn.example.net/example.org.hello.sxg
-      ```
```

Comment:
Since a self-signed certificate is used in this section, mentioning about `--ignore-certificate-errors-spki-list` would be still useful.

Reply:
Done.